src/utils.js

```diff
@@ -72,18 +73,15 @@ export function createEmptyAnchorElement( id, { writer } ) {
 /**
  * Creates an SVG placeholder {@link module:engine/view/emptyelement~EmptyElement} with the provided `id` attribute.
  *
- * @param {String} id
+ * @param {String} anchorId
  * @param {module:engine/conversion/downcastdispatcher~DowncastConversionApi} conversionApi
  * @returns {module:engine/view/emptyelement~EmptyElement}
  */
-export function createEmptyPlaceholderAnchorElement( id, { writer } ) {
-	const anchorElement = writer.createRawElement('span', { id }, function (domDocument) {
-       domDocument.innerHTML = `[INVISIBLE ANCHOR: ${id}]`;
-    });
-    writer.addClass("ck-anchor", anchorElement);
-	writer.setCustomProperty( 'anchor', true, anchorElement );
-
-	return anchorElement;
+export function createEmptyPlaceholderAnchorElement( anchorId, { writer } ) {
+	const anchorElement = writer.createContainerElement('span', {
+		class: 'ck-anchor-placeholder',
+	}, [writer.createText(`#${anchorId}`)]);
```

Comment:
Kepp the Invisible word
```
const anchorElement = writer.createContainerElement('span', {
     class: 'ck-anchor-placeholder',
}, [writer.createText(`[INVISIBLE ANCHOR: ${anchorId}]`)]);
```

Reply:
I think we should use CSS to add `[INVISIBLE ANCHOR: ` prefix and `]` suffix so people can override it easily. Perhaps I'll raise a new issue for that after reverting this.